panels/source/arrangers/Arranger.js

```diff
@@ -363,16 +363,20 @@
 				unit = unit || 'px';
 				if (!this.updating) {
 					// IE10 uses setBounds because of control hit caching problems seem in some apps
-					if (enyo.dom.canTransform() && !control.preventTransform && !enyo.platform.android && enyo.platform.ie !== 10) {
+					if (enyo.dom.canTransform() && !control.preventTransform && enyo.platform.ie !== 10) {
 						var l = bounds.left, t = bounds.top;
 						l = enyo.isString(l) ? l : l && (l + unit);
 						t = enyo.isString(t) ? t : t && (t + unit);
-						enyo.dom.transform(control, {translateX: l || null, translateY: t || null});
+						if (enyo.dom.canAccelerate()) {
```

Comment:
You guys are the experts. (I left enyo work awhile ago.) The point of this particular change was translateX/Y was slow and translate3d was fast. Any change that is library compatible and allows for acceleration would be an improvement... 

Not sure what design requirement came up with accelerated property and preventAccelerate. It'd be better to use the fast path of the device, and to know what it is instead of expecting programmers to code that. This stuff was all highly Android 2.3/2.2 related. i.e who cares now?